test/components/KonnectorTile.spec.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+/* eslint-env jest */
+
+import React from 'react'
+import { configure, shallow } from 'enzyme'
+import Adapter from 'enzyme-adapter-react-15'
+
+import { tMock } from '../jestLib/I18n'
+import { KonnectorTile } from 'components/KonnectorTile'
+
+configure({ adapter: new Adapter() })
```

Comment:
It is useful to configure the adapter in a jest setup file

Inside `package.json`:

```
"jest": {
  ...
  "setupFiles": [
    "./test/jest.setup.js"
  ]
}
```

```
import Adapter from 'enzyme-adapter-react-16'

configure({ adapter: new Adapter() })
```

Reply:
Yes, I already do that in cozy-store, but in the Home we have to change in all tests file so it has to be in a dedicated PR